conda-python-2.7.15-test-env.yaml

```diff
@@ -0,0 +1,175 @@
+name: pyrdf-dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - afterimage=1.21=hf014aae_1002
```

Comment:
What are all these packages? Are they all needed in the environment?

Reply:
ROOT's dependencies most of them. I'm using an environment file to speed up a bit the resolution of the dependencies since `conda` needs around 16min to calculate all of them.

Reply:
All those packages are the result of:

`conda install python=2.7.15 pytest pyspark`
`conda install -c conda-forge root`